docs/source/notebooks/tutorial.ipynb

```diff
@@ -4,40 +4,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tutorial"
+    "# Tutorial\n",
+    "\n",
+    "Computer simulations of the Earth’s climate and weather generate huge amounts of data. These data are often persisted on HPC systems or in the cloud across multiple data assets of a variety of formats (netCDF, zarr, etc…). Finding, investigating, loading these data assets into compute-ready data containers costs time and effort. The data user needs to know what data sets are available, the attributes describing each data set, before loading a specific data set and analyzing it. Intake ESM is a Python package that aims to provide a higher-level interface to searching and loading Earth System Model (ESM) data archives, such as CMIP6. "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Opening a collection\n",
-    "\n",
-    "First step is to point ``intake-esm`` to an ESM (Earth System Model) collection definition file, which is a JSON file that conforms to the [ESM Collection Specification](https://github.com/NCAR/esm-collection-spec). The collection JSON file can be stored on a local filesystem or can be hosted on a remote server. When provided a link/path to an esm collection file, `intake-esm` establishes a link to a database (CSV file) that contains assets (e.g. file) locations and associated metadata (i.e., which experiement, model, they come from). \n",
-    "\n",
-    "\n",
+    "## Loading a catalog\n",
     "\n",
-    "<div class=\"alert alert-info\">\n",
-    "\n",
-    "**Note:** \n",
-    "    \n",
-    "    \n",
-    "For demonstration purposes, we will be using the CMIP6 data hosted in Google Cloud Storage. For this data collection, we will be using the Pangeo cloud collection file stored [here](https://raw.githubusercontent.com/NCAR/intake-esm-datastore/master/catalogs/pangeo-cmip6.json).\n",
+    "Intake ESM works as follows:\n",
     "\n",
-    "    \n",
     "\n",
-    "More collection catalogs are listed [here](https://intake-esm.readthedocs.io/en/latest/faq.html#is-there-a-list-of-existing-catalogs). \n",
-    "    \n",
-    "    \n",
-    "</div>\n"
+    "1. Parse an ESM data catalog (stored in a JSON file). The JSON file must conform to the [ESM Collection Specification](https://github.com/NCAR/esm-collection-spec).\n",
```

Comment:
"1. Load an ESM data catalog file. The JSON-based catalog file must conform to the [ESM Collection Specification](https://github.com/NCAR/esm-collection-spec).\n",